### PR TITLE
CHE-13476 improve CHE-6 workspaces warning

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Improve CHE-6 workspaces warning. Add a link in the pop-up warning message to the documentation, change font icon.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13476

#### Docs PR

https://docs.google.com/document/d/1qrHg4F2FvTpafJAvgqgzeYTu6M0UA1mSjm_UCoQNFHc/edit?usp=sharing

Screenshot from 2019-06-10 19-06-04
https://youtu.be/2g9sdXLZQF8

### CODE REVIEW

**Comment by olexii4** [comment]
@slemeur  WDYT?

**Comment by olexii4** [comment]
ci-test

**Comment by olexii4** [comment]
ci-test

**Review by akurinnoy — APPROVED** [review]

File: dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts [hunk 1]
```diff
@@ -21,29 +25,47 @@ import {WorkspacesService} from '../../workspaces.service';
...
+              $sce: ng.ISCEService) {
     this.$location = $location;
     this.lodash = lodash;
     this.cheWorkspace = cheWorkspace;
     this.workspacesService = workspacesService;
+    this.$timeout = $timeout;
+    this.$document = $document;
     this.workspaceName = this.cheWorkspace.getWorkspaceDataManager().getName(this.workspace);
+
+    this.supportedRecipeTypeIsshue = $sce.trustAsHtml('Current infrastructure doesn\'t support this workspace recipe type.');
```
Comment (ashumilova) [inline dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts]: Isshue = Issue

Comment (olexii4) [reply dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts]: fixed. Thanks

**Comment by olexii4** [comment]
ci-test

**Review by ashumilova — APPROVED** [review]

**Comment by olexii4** [comment]
ci-test

**Comment by SkorikSergey** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1869//Selenium_20tests_20report/) doesn't show any regression against this Pull Request.

**Review by SkorikSergey — APPROVED** [review]

**Comment by slemeur** [comment]
@olexii4 What documentation it is pointing to?

**Comment by olexii4** [comment]
@slemeur  I have added the link

**Review by benoitf — APPROVED** [review]

**Comment by ashumilova** [comment]
Current state: waiting for the appropriate documentation to be merged.

File: dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts [hunk 2]
```diff
@@ -21,29 +25,47 @@ import {WorkspacesService} from '../../workspaces.service';
...
     this.cheWorkspace = cheWorkspace;
     this.workspacesService = workspacesService;
+    this.$timeout = $timeout;
+    this.$document = $document;
     this.workspaceName = this.cheWorkspace.getWorkspaceDataManager().getName(this.workspace);
+
+    this.supportedRecipeTypeIssue = $sce.trustAsHtml('Current infrastructure doesn\'t support this workspace recipe type.');
+
+    this.supportedVersionTypeIssue = $sce.trustAsHtml(`This workspace is using old definition format which is not compatible anymore. 
+          Please follow the <a href="${cheBranding.getDocs().workspace}" target="_blank">documentation</a>
```
Comment (l0rd) [inline dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts]: Would it be possible to reference the link to the doc section "Converting a Che 6 Workspace to a Che 7 devfile" ?

Comment (olexii4) [reply dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts]: Yes. It is possible. I have changed reference the link to the doc section "Converting a Che 6 Workspace to a Che 7 devfile"
https://github.com/eclipse/che/pull/13506/commits/2e2c2cc0aa78165f2b0b17da7b42ab370eb83f19

**Review by akurinnoy — APPROVED** [review]

File: dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts (hunk 2 above)
Comment (olexii4) [reply dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts]: done

**Review by l0rd — APPROVED** [review]

**Comment by l0rd** [comment]
Not in the endgame but necessary to deploy Che 7 on che.osio. I am +1 to merge to master.

**Comment by olexii4** [comment]
ci-test

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1967/) doesn't show any regression against this Pull Request.

**Pull request merged**